sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -87,7 +87,7 @@ describe("Create ServiceBusClient and Queue/Topic/Subscription Clients #RunInBro
   });
 });
 
-describe("Errors with non existing Namespace #RunInBrowser", function(): void {
+describe("Errors with non existing Namespace", function(): void {
```

Comment:
Add a TODO comment above to enable this for browser once Service Bus starts using core-amqp and a link to the github issue